pkg/extensions/meet/arcs/ExtApp.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright (c) 2022 Google LLC All rights reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file.
+ */
+import {
+  Paths, App, LocalStoragePersistor,
+  HistoryService,
+  ThreejsService,
+  ShaderService,
+  TensorFlowService,
+  SelfieSegmentationService,
+  MediaService
+} from './arcs.js';
+import {ExtRecipe} from './ExtRecipe.js';
+
+export const ExtApp = class extends App {
```

Comment:
wondering - this is the generic Extension app, right? or else what is meet specific here (except recipe, of course) ?

Reply:
At some level App is just a vehicle for configuration: the recipe, the services, the persistor, the Path maps.
This is an App that pulls those things together for the Meet experience I made.

There is also this, but it's not used right now:
`this.arcs.set('user', 'mediaDeviceState', {isCameraEnabled: true});`

Reply:
Is actually just a generic app. It has an `index.html` so you can run it without extension wrapper (but it doesn't do a whole lot unless you supply inputs).